mne/utils/docs.py

```diff
@@ -3495,6 +3553,50 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 For testing the lower tail (``tail=-1``), don't subtract ``pval`` from 1.
 """
 
+docdict['time_bandwidth_tfr'] = """
+time_bandwidth : float ``≥ 2.0``
```

Comment:
I'm surprised CIs don't complain about this.  In general we don't put the default on this line, but I'll admit it is concise to have it here instead of an extra sentence in the param description.

Reply:
I was surprised as well that it did not require an exclusion in `numpydoc_xref_ignore`!